tools/collect_metrics.py

```diff
@@ -79,99 +87,17 @@ class ConversionStatus(Enum):
-    return {
-        "ATen Input Variations": [x.schema for x in input_vars],
-        "Status": [string.capwords(x.status.name) for x in input_vars],
-    }
-
-
-def collect_input_variations_from_nodes(schemas: list):
-    """Creates a dictionary of unique nodes with their schema and input variations.
-
-    Returns:
-        ```
-        {
-            <opname>:
-            {
-                'opname': str,
-                'schema': {"args": list(tuple), "kwargs": list(tuple)}
-                'input_shapes': list(str),
-                'input_values': list(str|tuple),
-            },
-            <opname2>: {...},
-        }
-        ```
-
-    """
-    collection = {}
-    for node in schemas:
-        if "schema" in node:
-            opname = node["opname"]
-            input_shapes = node["input_shapes"]
-            input_values = node["input_values"]
-            # Create a new entry if opname has not been registered
-            if opname not in collection:
-                entry = {
-                    "opname": opname,
-                    "schema": node["schema"],
-                    "input_shapes": [input_shapes],
-                    "input_values": [input_values],
-                }
-                collection[opname] = entry
-            else:
-                if (
-                    input_shapes not in collection[opname]["input_shapes"]
-                    and input_values not in collection[opname]["input_values"]
-                ):
-                    collection[opname]["input_shapes"].append(input_shapes)
-                    collection[opname]["input_values"].append(input_values)
-    return collection
-
-
-def stringify_schema_to_string(schema, shapes, values, delimiter=", "):
-    """Combine schema types, shapes, names, and values to a single string."""
-    # holds each individual input
-    input_string_list = []
-    for i, (shape, value) in enumerate(zip(shapes, values)):
-        # This instance is a kwarg
-        if isinstance(value, tuple):
-            arg_name = value[0]
-            arg_type = schema["kwargs"][arg_name]
-            arg_val = f" = {value[1]}"
-        else:
-            arg_type = schema["args"][i][0]
-            arg_name = schema["args"][i][1]
-            arg_val = f" = {value}" if value else " = ?"
-        arg_shape = f"<{shape}>" if shape else ""
-        input_string_list.append(f"{arg_type}{arg_shape} {arg_name}{arg_val}")
-    string = delimiter.join(input_string_list)
-    return string
-
-
 def serialize_schema_metrics_to_operations(metrics):
     """Transform schema information to a list of `class Operation` pydantic models."""
     operations = []
     for node in metrics:
-        op_schema_string = stringify_schema_to_string(node["schema"], node["input_shapes"], node["input_values"])
-        operations.append(pydantic_models.Operation(op_name=node["opname"], op_schema=op_schema_string))
+        if "original_inputs" in node:
+            # This might not be right
```

Comment:
seems fine, why it might not be right?

Reply:
This is currently fine. The TTNN ops store the schema for the original aten ops in "original_inputs", so I wanted to remind myself to double check this. I think later if we want to save inputs for TTNN ops, we might slightly change this part.